README.md

```diff
@@ -145,24 +145,22 @@ Killgrave takes the following command line options. Killgrave is almost fully co
 ```sh
 $ killgrave -h
 
-  -config string
-        path with configuration file
-  -host string
-        if you run your server on a different host (default "localhost")
-  -imposters string
-        directory where your imposters are saved (default "imposters")
-  -port int
-        port to run the server (default 3000)
-  -secure bool
-        if you run your server using TLS (https)
-  -proxy-mode string
-        proxy mode you can choose between (all, missing or none) (default "none")
-  -proxy-url string
-        proxy url, you need to choose a proxy-mode
-  -version
-        show the _version of the application
-  -watcher
-        file watcher, reload the server with each file change
+Simple way to generate mock servers
+
+Usage:
+  killgrave [flags]
+
+Flags:
```

Comment:
Looks like you kept support for both short (e.g. `-c` or `-h`) and long (e.g. `--config` or `--help`) flags, which seems to be a great idea. However, by reading [the `v0.4.1` documentation](https://github.com/friendsofgo/killgrave/blob/v0.4.1/README.md) looks like the long flags were being provided with a single dash instead of double dash (note `-config` vs `--config`).

So, I'm wondering if it's only a documentation detail that needs to be fixed or if it's an actual breaking change still pending to be resolved.

Reply:
We need to make a decision here, because if we start to use `cobra` we will miss the single dash option: https://github.com/spf13/cobra/issues/740